Contributing-code-to-Oppia.md

```diff
@@ -18,6 +19,19 @@ If you run into any problems along the way, we're here to help! Check out our [[
 * [Tips for Success](#tips-for-success)
 * [Notes](#notes)
 
+## How to find information on this wiki
```

Comment:
Rename this title to something like "How to find information in the repo and the wiki" or "how to search for info on the repo".

Reply:
Done.